Spider.php

```diff
@@ -184,6 +184,7 @@ public function crawlUrl($uri)
         $crawler = $resource->getCrawler()->filterXPath('//a');
         $uris = array();
 
+        $this->persistenceHandler->setCoreName($this->getCoreName());
```

Comment:
I do not directly have a better alternative, but I don't feel really comfortable by the idea that the persistence handler has an idea about where stuff is being saved to. Currently we do use SOLR but that information should be irrelative for this service..